src/components/Metrics/Metrics.tsx

```diff
@@ -123,6 +143,11 @@ class Metrics extends React.Component<MetricsProps, MetricsState> {
     this.setState({ pollMetrics: newRefInterval });
   };
 
+  onReporterChanged = (reporter: string) => {
+    console.log('New reporter: ', reporter);
```

Comment:
Do you want this log ? or was just for debugging the feature ?

Reply:
It was for debugging. I'll remove it.